build/commands/lib/config.js

```diff
@@ -200,6 +228,15 @@ Config.prototype.enableCDMHostVerification = function () {
   return enable
 }
 
+Config.prototype.usePGO = function () {
+  // Disable PGO for non-public builds (i.e. local or PR's builds).
+  if (!this.isBraveReleaseBuild())
```

Comment:
This isn't quite what we talked about. We want to default `isBraveReleaseBuild` to true for release channels unless it is overridden by `is_brave_release_build`. Local builds won't have a channel set so `isBraveReleaseBuild` will be false and in PR builder we will set `is_brave_release_build` to false in npm config because they build as nightly channel.